packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -500,12 +504,17 @@ class TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
   }
 
   void _toggleNode(T node) {
+    //TODO: not a great way to check for null
+
+    if (node is! DominatorTreeNode) {
```

Comment:
Let's move the widget.onAnalysisRootChange to the function that is passed in called onToggleNode